charts/substra-backend/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.0
+
+- Added a new deployment: `substra-backend-events`
```

Comment:
I don't know if it's worth it to mention that a new config map was also added

Reply:
Good point! I don't think it's worth mentioning it because there's nothing particular to configure about it, from the point of view of a user